javaagent-tooling/src/main/java/io/opentelemetry/javaagent/tooling/ignore/GlobalIgnoredTypesConfigurer.java

```diff
@@ -55,7 +55,10 @@ private static void configureIgnoredTypes(IgnoredTypesBuilder builder) {
     // clojure
     builder.ignoreClass("clojure.").ignoreClass("$fn__");
 
-    builder.ignoreClass("io.opentelemetry.javaagent.");
+    // this is to exclude javaagent helper classes that are injected into other class loaders
+    // "io.opentelemetry.javaagent" classes in the bootstrap class loader are excluded elsewhere
+    // and all classes in the AgentClassLoader are excluded elsewhere
+    builder.ignoreClass("io.opentelemetry.javaagent.shaded.");
```

Comment:
Why is this here and not in `AgentInstaller`? I don't understand by what criterium we decide between these two locations.